### PR TITLE
Mes 5289 testing

### PR DESCRIPTION
**Pull request by naqvis5**
## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: test/e2e/pages/loginPage.ts [hunk 1]
```diff
@@ -20,29 +20,83 @@ class LoginPage extends Page {
...
+                  return authCContinue.click();
+                }
+                else {
+                  const acceptButton =   element(by.xpath(`//XCUIElementTypeButton[@name="Accept"]`));
+                  return acceptButton.click();
+                }
+              });
+          });
+
+        //
```
Comment (andrewsetterfield) [inline test/e2e/pages/loginPage.ts]: can this be removed.

File: test/e2e/pages/loginPage.ts [hunk 2]
```diff
@@ -20,29 +20,83 @@ class LoginPage extends Page {
...
+
+
 
         // Fill in password and click Sign in
-        const pFld = element(by.xpath(`//XCUIElementTypeOther[@name="Sign In"]/XCUIElementTypeOther[4]/XCUIElementTypeSecureTextField`));
-        browser.wait(ExpectedConditions.presenceOf(pFld));
-        pFld.sendKeys(password);
-        const signInButtonElement = element(by.xpath('//XCUIElementTypeButton[@name="Sign in"]'));
-        signInButtonElement.click();
+        // const pFld = element(by.xpath(`//XCUIElementTypeSecureTextField[@label="Enter the password for ${username}"]`));
```
Comment (andrewsetterfield) [inline test/e2e/pages/loginPage.ts]: why the commented out code in the code base?

File: test/e2e/pages/loginPage.ts [hunk 3]
```diff
@@ -70,10 +138,37 @@ class LoginPage extends Page {
...
+              selectUserToSignOut.click();
+
+              const cancelButton = element(by.xpath(`//XCUIElementTypeButton[@name="Cancel"]`));
+              browser.wait(ExpectedConditions.presenceOf(cancelButton));
+              cancelButton.click();
+
+              const signInLink = element(by.xpath(`//XCUIElementTypeStaticText[@name="Sign in"]`));
+              browser.wait(ExpectedConditions.presenceOf(signInLink));
+              signInLink.click();
+
```
Comment (andrewsetterfield) [inline test/e2e/pages/loginPage.ts]: why the big gap?

File: test/e2e/pages/searchPage.ts [hunk 4]
```diff
@@ -17,8 +17,8 @@ class SearchPage extends Page {
   }
 
   clickCloseButton() {
-    // this.clickElementByXPath('//div[2]/ion-header/ion-navbar/ion-buttons/button');
-    this.clickElementByCss('button.close-button-padding');
+    this.clickElementByXPath('//div//ion-header/ion-navbar/ion-buttons/button');
+    //this.clickElementByCss('button.close-button-padding');
```
Comment (andrewsetterfield) [inline test/e2e/pages/searchPage.ts]: commented out code

File: test/e2e/step-definitions/generic-steps.ts [hunk 5]
```diff
@@ -98,7 +98,8 @@ Given('I am not logged in', () => {
     // Switch to NATIVE context
     browser.driver.selectContext('NATIVE_APP').then(() => {
       // Wait until we are on the login page before proceeding
-      LoginPage.isCurrentPage();
+      // LoginPage.isCurrentPage();
```
Comment (andrewsetterfield) [inline test/e2e/step-definitions/generic-steps.ts]: commented out code

**Review by andrewsetterfield — CHANGES_REQUESTED** [review]

File: test/e2e/pages/searchPage.ts (hunk 4 above)
Comment (turnerc1) [inline test/e2e/pages/searchPage.ts]: Remove commented out code

File: test/e2e/pages/loginPage.ts [hunk 6]
```diff
@@ -70,10 +131,30 @@ class LoginPage extends Page {
...
+              cancelButton.click();
+
+              const signInLink = element(by.xpath(`//XCUIElementTypeStaticText[@name="Sign in"]`));
+              browser.wait(ExpectedConditions.presenceOf(signInLink));
+              signInLink.click();
+              // Switch back to WEBVIEW context
+              browser.driver.selectContext(this.getParentContext(webviewContext));
+            });
+            //     browser.driver.selectContext(this.getParentContext(webviewContext));
+            // browser.wait(ExpectedConditions.stalenessOf(element(by.className('click-block-active'))));
```
Comment (turnerc1) [inline test/e2e/pages/loginPage.ts]: remove comments

**Review by turnerc1 — CHANGES_REQUESTED** [review]

**Review by turnerc1 — APPROVED** [review]

File: test/e2e/pages/loginPage.ts (hunk 1 above)
Comment (naqvis5) [reply test/e2e/pages/loginPage.ts]: done

File: test/e2e/pages/loginPage.ts (hunk 2 above)
Comment (naqvis5) [reply test/e2e/pages/loginPage.ts]: done

File: test/e2e/pages/loginPage.ts (hunk 3 above)
Comment (naqvis5) [reply test/e2e/pages/loginPage.ts]: done

File: test/e2e/pages/searchPage.ts (hunk 4 above)
Comment (naqvis5) [reply test/e2e/pages/searchPage.ts]: done

File: test/e2e/step-definitions/generic-steps.ts (hunk 5 above)
Comment (naqvis5) [reply test/e2e/step-definitions/generic-steps.ts]: done

File: test/e2e/pages/searchPage.ts (hunk 4 above)
Comment (naqvis5) [reply test/e2e/pages/searchPage.ts]: done

File: test/e2e/pages/loginPage.ts (hunk 6 above)
Comment (naqvis5) [reply test/e2e/pages/loginPage.ts]: done

**Review by andrewsetterfield — APPROVED** [review]

**Pull request merged**